src/modules/site/site.service.test.ts

```diff
@@ -11,9 +12,9 @@ jest.mock('../graph/graph.service');
 describe('SiteService', () => {
   const valueGenerator = new RandomValueGenerator();
 
-  const ukefSharepointName = valueGenerator.string();
-  const tfisSiteName = valueGenerator.string();
-  const tfisListId = valueGenerator.string();
+  const ukefSharepointName = ENVIRONMENT_VARIABLES.SHAREPOINT_MAIN_SITE_NAME;
+  const tfisSiteName = ENVIRONMENT_VARIABLES.SHAREPOINT_TFIS_SITE_NAME;
+  const tfisListId = ENVIRONMENT_VARIABLES.SHAREPOINT_TFIS_LIST_ID;
```

Comment:
I don't think we want to make this change for the unit tests - the unit tests should be making sure that the service gets these values from the constructor, but since we're using special values now we don't know if the service is using the values from the constructor or if it is importing the constants from a different file

Reply:
Nice nice -- I've updated this. Needed to update the generator as a result too :)